### ISSUE TITLE
JSON parse errors: error.expected.long()

### ISSUE THREAD

**Issue by PinGwynn** [adobe-research/prometheus-opentsdb-exporter#7]
Hello,

I've observed errors in my logfile with messages like:
`prometheus-opentsdb-exporter_1  | [error] application - JSON parse errors: [{"path":"(29)/dps/1580218670","errors":["error.expected.long()"]},{"path":"(29)/dps/1580246690","errors":["error.expected.long()"]},{"path":"(57)/dps/1580209790","errors":["error.expected.long()"]}]
`
When I tried to handle what is happening, I found, that the exporter expects Long type value in dps, but it is not mandatory for OpenTSDB. For example, I'm getting float values (which is coused such errors):
```
    "dps": {
      "1580205650": 123717,
      "1580205710": 123724,
      "1580218670": 124447.5,
      "1580218730": 124452,
}
```

**Comment by andreifecioru** [comment]
Thank you for the bug report. Will take a look as soon as possible.

**Comment by andreifecioru** [comment]
I have published a new minor release (v0.3.3) which attempts to fix this issue (see commit https://github.com/adobe-research/prometheus-opentsdb-exporter/commit/021b8daddba7f58dadfb8021dc2c655d72da4e8e).

Please let me know if this works on your side.
